iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothubservices/telemetry/SendMessagesTests.java

```diff
@@ -43,6 +43,20 @@ public SendMessagesTests(IotHubClientProtocol protocol, AuthenticationType authe
         super(protocol, authenticationType, clientType, publicKeyCert, privateKey, x509Thumbprint, withProxy);
     }
 
+    @Test
+    public void sendSecurityMessages() throws Exception
+    {
+        if (testInstance.protocol == MQTT_WS && (testInstance.authenticationType == SELF_SIGNED || testInstance.authenticationType == CERTIFICATE_AUTHORITY))
+        {
+            //mqtt_ws does not support x509 auth currently
+            return;
+        }
+
+        this.testInstance.setup();
+
+        IotHubServicesCommon.sendMessages(testInstance.client, testInstance.protocol, NORMAL_MESSAGES_TO_SEND, RETRY_MILLISECONDS, SEND_TIMEOUT_MILLISECONDS, 0, null);
+    }
+
```

Comment:
This looks like the same test as below, so how does this test sending a security message?

Reply:
Yes, it should be sendSecurityMessages call.

---
In reply to: [321013544](https://github.com/Azure/azure-iot-sdk-java/pull/576#discussion_r321013544) [](ancestors = 321013544)